apps/laboratory/app/library/exclude-wallet-ids/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
```

Comment:
nit: think it would be nice to start putting these things inside `laboratory/app/flags` instead

At some point we need to move a lot of views out oof `library`. `library` should be to test specifics of libraries/adapters imo

Reply:
Totally agree. Let's start doing that actually, will push now